### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pogman**
TL;DR - Auto scrolling log + Comms Errors (none at all for me now!!!)

### CODE REVIEW

File: app/src/main/java/info/nightscout/android/medtronic/MainActivity.java [hunk 1]
```diff
@@ -363,6 +382,9 @@ protected void onStart() {
     @Override
     protected void onPostCreate(Bundle savedInstanceState) {
         super.onPostCreate(savedInstanceState);
+//        sendStatus("Nightscout 600 Series Uploader");
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/MainActivity.java]: I prefer not to include commented code.

File: app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlAlarmManager.java [hunk 2]
```diff
@@ -66,8 +66,13 @@ public static void setAlarm(long millis) {
 
     // restarting the alarm after MedtronicCnlIntentService.POLL_PERIOD_MS from now
     public static void restartAlarm() {
-        //setAlarmAfterMillis(MainActivity.pollInterval + MedtronicCnlIntentService.POLL_GRACE_PERIOD_MS);
-        setAlarmAfterMillis(ConfigurationStore.getInstance().getPollInterval()); // grace already accounted for when using current intent time to set default restart
+//        setAlarmAfterMillis(ConfigurationStore.getInstance().getPollInterval()); // grace already accounted for when using current intent time to set default restart
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlAlarmManager.java]: Do we need this commented code at all?

File: app/src/main/java/info/nightscout/android/medtronic/MainActivity.java (hunk 1 above)
Comment (Pogman) [inline app/src/main/java/info/nightscout/android/medtronic/MainActivity.java]: OK - that was WIP as unsure whether to notify user of poll intervals on startup. Will leave this kind of stuff out.

File: app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlAlarmManager.java (hunk 2 above)
Comment (Pogman) [inline app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlAlarmManager.java]: Not really. It's already noted in the advice comment.

**Pull request merged**